src/Sylius/Bundle/ShopBundle/Resources/views/Cart/Block/_cartButton.html.twig

```diff
@@ -0,0 +1,7 @@
+{% import "@SyliusShop/Common/Macro/money.html.twig" as money %}
```

Comment:
Why the name of this template is `_cartButton`? IMO you should add somewhere that this is a widget